### PR TITLE
Document the geo location support on Android SDK

### PR DESCRIPTION
**Pull request by ben181231**
connect SkygearIO/skygear-SDK-Android#56

### CODE REVIEW

**Comment by oursky-travis** [comment]
@ben181231, thanks for your PR! By analyzing the annotation information on this pull request, we identified @HenryHins and @bensonby to be potential reviewers

File: content/android/geolocation.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Skygear supports saving Geo Location as a field in record. Please be reminded
+that Geo Location is identified by latitude and longitude.
+
+### Saving a location in a record
+
+```java
+Location loc = /* get the location via LocationServices or create a new one */
```
Comment (bensonby) [inline content/android/geolocation.md]: 1. Can we have a sample location here?
2. Is this `Location` class https://developer.android.com/reference/android/location/Location.html ?

File: content/android/geolocation.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Record aRecord = new Record("Note");
+aRecord.set("place", loc);
+
+skygear.getPublicDatabase().save(aRecord, new RecordSaveResponseHandler(){
+    /* Implementation of response handler */
```
Comment (bensonby) [inline content/android/geolocation.md]: Will it be too complicated to have a quick implementation showing the latitude of the saved record?

Comment (ben181231) [inline content/android/geolocation.md]: It is not so complicated. Let me add it here.

File: content/android/geolocation.md (hunk 1 above)
Comment (ben181231) [inline content/android/geolocation.md]: > Can we have a sample location here?

Do you mean I directly create a `Location` instance by latitude and longitude ?

> Is this Location class https://developer.android.com/reference/android/location/Location.html ?

Yes.

**Comment by ben181231** [comment]
@bensonby Updated.

File: content/android/geolocation.md (hunk 1 above)
Comment (bensonby) [inline content/android/geolocation.md]: Yes, I mean creating a simple `Location` instance

**Comment by ben181231** [comment]
@bensonby Updated again. 😅

**Comment by bensonby** [comment]
I added some commit and made a new PR #359

**Pull request closed without merging**